service_info_cms/cms_menus.py

```diff
@@ -25,4 +25,23 @@ def modify(self, request, nodes, namespace, root_id, post_cut, breadcrumb):
             self._modify_nodes(nodes)
         return nodes
 
+
+class AddRatingExtension(Modifier):
+    """Make the RatingExtension available in the menu."""
```

Comment:
I thought this was the code that put the "Allow page ratings..." setting into the drop down menu?